### PR TITLE
Implement function to initialize rag search pipeline

### LINKED ISSUE #82: [FEATURE][DocBot] Add RAG Search Pipeline
### Is your feature request related to a problem?
We need a RAG Search Pipeline that will be used with our `cohere-index`. 

### What solution would you like?
In our cluster_bootstrap.py we need to create a search pipeline that will be used as the [default](https://opensearch.org/docs/latest/search-plugins/search-pipelines/using-search-pipeline/#default-search-pipeline) for our `cohere-index`. This search pipeline will need to be created after our models are initiated and before the cohere-index is created. 

We will need to use the new LLM_MODEL_ID from our model state as the model for this. See #62

Here is the reference for creating a RAG search pipeline in OpenSearch: https://opensearch.org/docs/latest/ml-commons-plugin/conversational-search/#setting-up-the-pipeline

### PR DESCRIPTION
**Pull request by ortegaa32**
### Description
Adds method to initialize the RAG search pipeline as well as making it the default for the `cohere-index`.

### Issues Resolved
Resolves #82 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: demo/docbot/cluster_bootstrap.py [hunk 1]
```diff
@@ -242,7 +242,35 @@ def initialize_ingestion_pipeline(client: MLClient):
...
+    Returns:
+        returns None if pipeline was created or it exists else raises Exception
+    """
+    body = {
+      "response_processors": [
+        {
+          "retrieval_augmented_generation": {
+            "description": "RAG search pipeline to be used with Cohere index",
+            "model_id": MODEL_STATE["LLM_model_id"],
+            "context_field_list": ["text"]
```
Comment (dtaivpp) [inline demo/docbot/cluster_bootstrap.py:258]: For context field list I think we are looking for our `content` filed here

**Review by dtaivpp — CHANGES_REQUESTED** [review]
One change and I think this will be good!

**Comment by ortegaa32** [comment]
Just made the relevant change

**Review by dtaivpp — APPROVED** [review]
Awesome! Thanks :D

**Pull request merged**